tests/scaleway/scw_container_registry.rs

```diff
@@ -0,0 +1,254 @@
+extern crate test_utilities;
```

Comment:
I'm not sure those tests are useful as they will be validated during applications tests deployments

Reply:
Indeed, I agree, I will put an ignore once all functional tests will be up and running. But still, might be useful when functional tests will be broken to help investigate more in depth.
Let me know if ignore is ok for you ?

Reply:
Done, I added ignore for those kind of tests